apps/frontend/src/components/global/ExportHTMLModal.vue

```diff
@@ -313,10 +325,15 @@ export default class ExportHTMLModal extends Vue {
   iconDataToSVG(
     iconData: string,
     fill: string,
+    description = 'None',
     widthPx = 24,
     heightPx = 24
   ): string {
-    return `<svg style="width:${widthPx}px; height:${heightPx}px" viewBox="0 0 ${widthPx} ${heightPx}" role="img" aria-hidden="true"><path fill="${fill}" d="${iconData}"/></svg>`;
+    if (description === 'None') {
+      return `<svg style="width:${widthPx}px; height:${heightPx}px" viewBox="0 0 ${widthPx} ${heightPx}" role="img" aria-hidden="true"><path fill="${fill}" d="${iconData}"/></svg>`;
```

Comment:
could probably simplify this so we don't need a wrapping conditional, or at least not one with as much duplication as is going on here (DRY etc)

```
const label = ` aria-label="${description}"`;
return `<svg style="width:${widthPx}px; height:${heightPx}px" viewBox="0 0 ${widthPx} ${heightPx}" role="img" title="${description}"${description ? label : ''}><path fill="${fill}" d="${iconData}"/></svg>`;
```

Reply:
Changed as requested.